dev/sparktestsupport/toposort.py

```diff
@@ -0,0 +1,85 @@
+#######################################################################
+# Implements a topological sort algorithm.
```

Comment:
Oops, I missed this. It looks like it's Apache licensed and you have correctly preserved the header. https://bitbucket.org/ericvsmith/toposort/src/25b5894c4229cb888f77cf0c077c05e2464446ac/LICENSE.txt?fileviewer=file-view-default

You'll need to put the `NOTICE` contents in our `NOTICE`:
https://bitbucket.org/ericvsmith/toposort/src/25b5894c4229cb888f77cf0c077c05e2464446ac/NOTICE?fileviewer=file-view-default

That should be all.